src/python/WMCore/Cache/WMConfigCache.py

```diff
@@ -322,7 +323,8 @@ def loadByView(self, view, value):
 
         if len(viewRes['rows']) == 0:
             # Then we have a problem
-            logging.error("Unable to load using view %s and value %s" % (view, str(value)))
```

Comment:
This will stop the warning, but what it's actually asking you to do is 
```
logging.error('msg %s %s', view, value)
```
instead of 
```
logging.error('msg %s %s' % (view, value)'
```
The first way performs better by not having to construct the strings when the message will be dropped (especially good for debug messages).